README.md

```diff
@@ -130,6 +130,7 @@ end.
 
    ```
    cd azul
+   source environment
```

Comment:
Oops. This order won't work. `source environment` calls `scripts/export_environment.py` which requires the symlink `.active` (step 3) has been created.

Reply:
Ah, my bad. Fixed this.